src/app/components/popup-host/popup-host.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="modal" [ngClass]="{modal_open: openDialog()}">
```

Comment:
Я всегда рекомендую подход с биндингом классов, он читается как то попроще и биндинг более нативная история, Ангуляру чуть проще ее оптимизировать и мигрировать на новые инструменты
```suggestion
<div class="modal" [class. modal_open]="openDialog()">
```

Reply:
спасибо, буду иметь ввиду!